web/app/ui/CreateMetalakeDialog.js

```diff
@@ -37,7 +37,13 @@ const defaultValues = {
 }
 
 const schema = yup.object().shape({
-  name: yup.string().required()
+  name: yup
```

Comment:
I think you should change the catalog, schema, table, and other name identifier-related string also.

Reply:
This `.js` file only works for managing the catalog.
Does the `key` and `value` of field `properties` need this rules?
I will add the same rules when managing the schema/table.

Reply:
> Does the key and value of field properties need this rules?
I will add the same rules when managing the schema/table.

Yes, I think so, we'd better have a basic check for kv fields, not just the name.

Besides, I think we should also have a name check in the backend, not just in the frontend. This can be done is the next version. @yuqi1129 can you please create an issue to track the backend work?

Reply:
> Besides, I think we should also have a name check in the backend, not just in the frontend. This can be done is the next version. @yuqi1129 can you please create an issue to track the backend work?

Got it.